snuba/clickhouse/columns.py

```diff
@@ -69,7 +73,7 @@ class ColumnSet(BaseColumnSet):
     def __init__(
         self,
         columns: Sequence[
-            Union[Column[SchemaModifiers], Tuple[str, ColumnType[SchemaModifiers]]]
+            Union[Column[SchemaModifiers], PTuple[str, ColumnType[SchemaModifiers]]]
```

Comment:
You actually don't need to import `Tuple` from typing if you have the `future` import in the file. You can type things with the builtin Python type. This is true for list, dict etc. https://www.python.org/dev/peps/pep-0585/
There are a couple edge cases where that isn't true but not here.

```suggestion
            Union[Column[SchemaModifiers], tuple[str, ColumnType[SchemaModifiers]]]
```

Reply:
Removed all aliases in favor of the built-in type.